pgcli/main.py

```diff
@@ -71,7 +91,16 @@ def cli(database, user, password, host, port):
 
     # Connect to the database.
     try:
-        pgexecute = PGExecute(database, user, passwd, host, port)
+        try:
+            pgexecute = PGExecute(database, user, passwd, host, port)
+        except OperationalError as e:
+            if 'no password supplied' in e.message and auto_passwd_prompt:
```

Comment:
Once again a comment here would be preferable, to say that a connection retry is attempted since a password was required by wasn't provided.